### PR TITLE
delete comment handling

### PR DESCRIPTION
**Pull request by megaherz**

### CODE REVIEW

File: event/events.go [hunk 1]
```diff
@@ -38,6 +39,7 @@ func ToEvent(op types.Operation, blockID string, blockNum uint32, timestamp time
 	return commonEvent
 }
 
+// Common Event
```
Comment (cali4888) [inline event/events.go]: why do we need those comments

File: event/events.go [hunk 2]
```diff
@@ -211,3 +218,31 @@ func toCommentEvent(op types.Operation, commonEvent CommonEvent) Event {
...
+}
+
+func (e DeleteCommentEvent) Common() CommonEvent {
+	return e.CommonEvent
+}
+
+func toDeleteCommentEvent(op types.Operation, commonEvent CommonEvent) Event {
+	v, ok := op.(*types.DeleteCommentOperation)
+	if !ok {
+		panic(errWrongEventType)
```
Comment (cali4888) [inline event/events.go]: is it really good idea to panic in a library?

File: event/events.go (hunk 1 above)
Comment (megaherz) [reply event/events.go]: Simply for navigation

Comment (cali4888) [reply event/events.go]: So if we are leaving such comments maybe we can follow Golint format for them? `CommonEvent` instead `Common Event`

File: provider/provider_test.go [hunk 3]
```diff
@@ -25,8 +25,8 @@ func TestProvider(t *testing.T) {
 		})
 
 	select {
-	case <- time.Tick(1 * time.Second):
-		t.Fail()
+	case <- time.Tick(5 * time.Second):
```
Comment (cali4888) [inline provider/provider_test.go]: actually, why do we have time based test cases here? It's not very obvious for me. Maybe leave a comment here.

File: event/events.go (hunk 2 above)
Comment (megaherz) [reply event/events.go]: This panic should never happen. If it happens the developer of the library made a mistake.

File: event/events.go (hunk 1 above)
Comment (megaherz) [reply event/events.go]: Good point, done

**Pull request merged**